moduleroot/.travis.yml.erb

```diff
@@ -106,11 +106,52 @@ matrix:
       <%= key %>: <%= job[key].gsub(/@@SET@@/, set['set']).gsub(/@@COLLECTION@@/, set.fetch('collection', 'puppet6')).gsub(/@@TESTMODE@@/, set.fetch('testmode', 'apply')) %>
 <%   end -%>
 <% end -%>
+<%#The following variables are used when use_litmus_travis is enabled.-%>
+<% 
+  if @configs['puppet_collection'].nil? 
```

Comment:
If these are only used in the `use_litmus_travis ` context, then may as well put them inside the `if` statement below